airgun/views/host.py

```diff
@@ -39,26 +40,68 @@ class TableActions(View):
 class PuppetClassParameterValue(Widget):
     """Represent value field for Puppet Class parameters table row from Host
     Parameters tab. That field can be interacted with as usual text input, but
-    also it can be overriden with new value
+    also it can be overridden with new value
     """
     ROOT = ".//div[@class='input-group']"
     value = TextInput(
-        locator=".//textarea[contains(@name, 'lookup_values_attributes')]")
-    override_parameter = Text(".//a[@data-tag='override']")
-    remove_override = Text(".//a[@data-tag='remove']")
+        locator=".//*[self::textarea or self::input][contains(@name, 'lookup_values_attributes')]")
+    override_button = Text(".//a[@data-tag='override']")
+    remove_override_button = Text(".//a[@data-tag='remove']")
+    hide_button = Text(".//a[contains(@class, 'btn-hide')]")
+
+    def _button_is_displayed(self, button_widget):
+        """Return whether the widget is displayed"""
+        return self.browser.wait_for_element(
```

Comment:
maybe use selenium `is_displayed` instead?

Reply:
I'am not sure if browser.is_displayed will raise exception in case of not visible as we need ti check_visibility

Reply:
any way will test that and see